site/gatsby-site/playwright/e2e-full/apps/submitted.spec.ts

```diff
@@ -326,6 +326,7 @@ test.describe('Submitted reports', () => {
             editor_notes: "",
             description: 'Sarasa',
             title: "",
+            implicated_systems: ["entity-1"],
```

Comment:
could you please add this field to the query below and check for it being returned? (IIUC, we're adding the field to the DB seed but not testing it.)

Reply:
Noted! I added this check to another test